### PR TITLE
katello-host-tools 3.3.6 release

### PR DESCRIPTION
**Pull request by jlsherrill**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by jturel** [comment]
Might need to add a check for zypper_install here too:

https://github.com/theforeman/foreman-packaging/blob/acc5bf86445a0ec7a8caada1d93a75a60625bafc/packages/client/katello-host-tools/katello-host-tools.spec#L184

**Comment by jturel** [comment]
Here too

https://github.com/theforeman/foreman-packaging/blob/acc5bf86445a0ec7a8caada1d93a75a60625bafc/packages/client/katello-host-tools/katello-host-tools.spec#L301

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 1]
```diff
@@ -179,7 +181,7 @@ rm %{buildroot}%{katello_libdir}/agent/goferd/legacy_plugin.py
 %endif
 %endif
 
-%if %{dnf_install} || %{yum_install}
+%if %{dnf_install} || %{yum_install} || %{zypper_install}
```
Comment (ehelms) [inline packages/client/katello-host-tools/katello-host-tools.spec]: Is there a build that does not trigger one of these at this point?

File: packages/client/katello-host-tools/katello-host-tools.spec [hunk 2]
```diff
@@ -296,7 +298,7 @@ exit 0
 %doc LICENSE
 %dir %{_localstatedir}/cache/katello-agent/
 
-%if %{dnf_install} || %{yum_install}
+%if %{dnf_install} || %{yum_install} || %{zypper_install}
```
Comment (ehelms) [inline packages/client/katello-host-tools/katello-host-tools.spec]: Is there a build that does not trigger one of these at this point?

File: packages/client/katello-host-tools/katello-host-tools.spec (hunk 1 above)
Comment (jlsherrill) [reply packages/client/katello-host-tools/katello-host-tools.spec]: There isn't now, but dnf_install will be removed as part of the 3.4 release, so it seemed simpler to just add 1 condition and remove one later than remove the entire check

File: packages/client/katello-host-tools/katello-host-tools.spec (hunk 2 above)
Comment (jlsherrill) [reply packages/client/katello-host-tools/katello-host-tools.spec]: same as above

**Comment by jlsherrill** [comment]
@ehelms not really, 3.10 is fine

**Pull request merged**